fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -1906,6 +1906,160 @@ countries:
                 issue_months: [0, 1, 2, 3, 4, 5]
                 start_year: 1991
 
+    mali:
+        logo: Mali_IRI_120x48.png
+        resolution: [0.05, 0.05]
+        center: [-4.0, 17.0]
+        zoom: 6
+        rotation: 0
+        marker: [-4.025, 17.025]
+        shapes:
+            - name: National
+              sql: select adm0_code as key, adm0_name as label, ST_AsBinary(the_geom) as the_geom
+                  from iridb.g2015_2012_0 where adm0_name = 'Mali'
+              vuln_sql: select adm0_code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Mali' group by 1, 2
+
+            - name: Regional
+              sql: select (adm0_code, adm1_code) as key, adm1_name as label,
+                  ST_AsBinary(the_geom) as the_geom from iridb.g2015_2014_1
+                  where adm0_name = 'Mali'
+              vuln_sql: select (adm0_code, adm1_code) as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Mali' group by 1, 2
+
+            - name: Department
+              sql: select (adm0_code, adm1_code, adm2_code) as key, adm2_name as label,
+                  ST_AsBinary(the_geom) as the_geom from iridb.g2015_2014_2
+                  where adm0_name = 'Mali'
+              vuln_sql: select (adm0_code, adm1_code, adm2_code) as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Mali' group by 1, 2
+
+        datasets:
+            defaults:
+              predictors:
+                - prcp-v1
+                - chirps-precip-jas
+              predictand: bad-years-emdat
```

Comment:
This has to refer to  one of the obs datasets, but there's no dataset with this name. In line 2027 you've named it `bad-years`.